pkgs/applications/graphics/xournal/default.nix

```diff
@@ -47,11 +49,11 @@ stdenv.mkDerivation rec {
       cp $out/share/xournal/pixmaps/xournal.png $out/share/icons
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     homepage = http://xournal.sourceforge.net/;
     description = "Note-taking application (supposes stylus)";
-    maintainers = [ stdenv.lib.maintainers.guibert ];
-    license = stdenv.lib.licenses.gpl2;
-    platforms = stdenv.lib.platforms.linux;
+    maintainers = [ maintainers.guibert ];
+    license = licenses.gpl2;
+    platforms = with platforms; [ linux darwin ];
```

Comment:
Attributes in platforms [are lists](https://github.com/NixOS/nixpkgs/blob/master/lib/platforms.nix) so this will result in nested lists: `[ [ "i686-linux" ... ] [ "x86_64-darwin" ] ]`